### ISSUE TITLE
Bug for Internet Explorer

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#179]
In GitLab by @olivierbur on Aug 19, 2019, 13:42

In Internet Explorer, the boxes don't open, if clicked upon. 

I.e. https://selfhelp.psy.unibe.ch/hermes/WasistHERMES? 

" Ist HERMES etwas für mich? (Hier klicken) "

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Aug 20, 2019, 14:15

The problem stems from the new style `mermaidForm`. I now learned that [mermaid does not support IE](https://github.com/knsv/mermaid/issues/303) (which makes sense, Edge replaced IE four years ago and is now shipped with Windows by default).

Of course, mermaid not working on IE should not break the rest of the page. I fixed this by loading the mermaid file separately and not as part of the style package. Mermaid will still fail but it will not drag down the rest of the scripts: 587539c
